packages/babel-plugin-polyfill-corejs3/src/built-in-definitions.ts

```diff
@@ -20,7 +20,7 @@ const define = (
   pure,
   global,
   name = global[0],
-  exclude,
+  exclude?,
```

Comment:
similar to the other file, this threw a TS error as it wasn't optional before, but is not always used